spec/lib/telephony/otp_sender_spec.rb

```diff
@@ -255,4 +255,100 @@
       end
     end
   end
+
+  describe '#authentication_message' do
+    let(:sender) do
+      sender = Telephony::OtpSender.new(
+        to: '+18888675309',
+        otp: 'ABC123',
+        channel: 'sms',
+        expiration: TwoFactorAuthenticatable::DIRECT_OTP_VALID_FOR_MINUTES,
+        domain: 'secure.login.gov',
+        country_code: 'US',
+      )
+    end
+
+    context 'sms' do
+      context 'English' do
+        it 'does not contain any non-GSM characters and is less than or equal to 160 characters' do
+          message = sender.authentication_message
+          expect(Telephony.sms_parts(message)).to eq 1
+          expect(message.chars.all? { |x| Telephony::GSM_CHARACTERS.include?(x) }).to eq true
+        end
+      end
+
+      # The Spanish-language translation currently includes the 'ó', character, which is not
+      # in the GSM 03.38 character set
+      context 'Spanish' do
+        it 'is sent in three parts' do
+          I18n.locale = :es
+
+          message = sender.authentication_message
+          expect(Telephony.sms_parts(message)).to eq 3
+        ensure
+          I18n.locale = :en
+        end
+      end
```

Comment:
our goal is that this could be a 1-message text right? what if we asserted that and made it a `pending` spec?, ex:

```suggestion
      context 'Spanish' do
        it 'is sent in one part' do
          I18n.locale = :es

          pending 'spanish translations that do not use ó (outside GSM 03.38 charset)'

          message = sender.authentication_message
          expect(Telephony.sms_parts(message)).to eq 1
        ensure
          I18n.locale = :en
        end
      end
```

Reply:
My goal was to make it the expectations explicit so we can detect if we go from 3 to 4. I don't think we'll ever get Spanish down to 1 part with the constraints as they are.

Reply:
I feel like if this ever got to be high-enough volume we could just remove the "ó" and turn it into an "o", it obviously wouldn't be ideal but the core message would still get across

Reply:
Yep, it's definitely an option. I think @pauldoomgov said we are more than ok with sending more parts as long as it's for a good reason (a more correct translation in this case).